pom.xml

```diff
@@ -256,6 +256,12 @@
                 <version>1.1.1.6</version>
             </dependency>
 
+            <dependency>
+                <groupId>io.dropwizard.metrics</groupId>
+                <artifactId>metrics-graphite</artifactId>
+                <version>4.0.5</version>
```

Comment:
Our current metrics version in geomesa is set to 3.1.2. We were discussing updating it. Is this going to cause compatibility issues? cc @jahhulbert-ccri